### ISSUE TITLE
Not working in xfce4-terminal

### ISSUE THREAD

**Issue by J-Moravec** [veloce/vim-aldmeris#3]
Hullo, big fan of the Oblivion color scheme, so I have been using Aldmeris for a while.

Recently, I have moved from gnome-terminal to xfce4-terminal, but the color scheme doesn't work.

It works fine in gvim, but in xfce4-terminal, the behaviour is the same as in xterm. So I would guess the script doesn't recognize xfce4-terminal and treats it as less capable terminal compared to gnome-terminal?

**Comment by veloce** [comment]
I think it related to the term color palette, you should check that.

**Comment by J-Moravec** [comment]
Removing the `let g:aldmeris_termcolors = "tango"`  setting helped.

I got weird behaviour when the same .vimrc produced the desired colours on a remote machine running `vim` inside `tmux`. But not when running vim on a remote machine in a terminal or locally in a terminal.

So I played a bit with colours and found out that removing the above setting makes the colours consistent.

Maybe add a warning to the documentation @veloce ?
